src/app/admin/adminActions.ts

```diff
@@ -17,3 +20,16 @@ export const deleteUser = async (userId: string): Promise<FunctionsResponse<any>
         body: JSON.stringify({ userId }),
     });
 };
+
+interface UpdateUserRequestBody {
+    userId: string;
+    attributes: AdminUserAttributes;
+}
+
+export const updateUser = async (
+    userDetails: UpdateUserRequestBody
+): Promise<FunctionsResponse<any>> => {
```

Comment:
What does the type parameter in `FunctionResponse` do? Is there a way to get rid of the any (unknown is usually safer if you can't specify a type)

Reply:
Apparently this is just the type that gets returned by `functions.invoke`